app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -62,100 +62,85 @@ You’ll earn a [competitive salary](/salaries-and-benefits), get supported in y
 
-* teaching skills and experience
+* to apply for English 'qualified teacher status', or be able to demonstrate strong non-UK qualifications and experience 
+* a high standard of written and spoken English
 * to pass criminal and professional safeguarding checks (these will be organised by your employer)
-* to [get the right visa or status](#get-the-right-visa-or-status)
+* a visa or immigration status allowing you to work in the UK
 
-It is also helpful to have a teaching qualification from your own or another non-UK country, or to get '[qualified teacher status](#consider-getting-qualified-teacher-status-qts)'.
+### Your professional qualifications 
 
-## Get the right visa or status
+Schools you apply to will want to see evidence of your teaching and academic qualifications. [UK ENIC can give you a statement of comparability](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) showing how your qualifications compare to English ones. 
 
-If you’re not sure about your immigration status, you can [check if you need a UK visa](https://www.gov.uk/check-uk-visa). You can also [contact UK Visas and Immigration for help](https://www.gov.uk/contact-ukvi-inside-outside-uk).
+## Apply for qualified teacher status (QTS)
 
-You can apply for a skilled worker visa once you have been offered a teaching job at certain schools in England. [Check our guidance on visas and immigration](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration) to understand which schools can sponsor a skilled worker visa. 
-
-You may also be eligible for another type of visa, or you may have a pre-existing right to work in the UK, such as settled or pre-settled status under the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families) or [indefinite leave to remain in the UK](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk).
-
-[If you're an Irish citizen you do not need a visa](https://www.gov.uk/government/publications/common-travel-area-guidance).
-
-## Consider getting qualified teacher status (QTS)
-
-[Qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) is the professional status teachers in England gain at the end of their teacher training.
+[Qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) is the professional status teachers in England gain at the end of their teacher training. 
 
 If you’re a qualified teacher from outside the UK, you can [work as a teacher in England for up to 4 years without QTS](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk#qts-exemption-for-teachers-from-outside-the-uk).
```

Comment:
Is this link needed?

Reply:
Yes I think so to explain the 4 year rule